README.Rmd

```diff
@@ -0,0 +1,114 @@
+kable(dad_mom, format = "markdown")
+### Just taking a look at what we're working with
+```
+
+```{r tidying, echo = FALSE}
+tidy_dad_mom <- dad_mom %>%
+  unite(dad, name_dad, income_dad) %>%
+  unite(mom, name_mom, income_mom) %>%
+### Put together all the mom data under 'mom' and dad data under 'dad', now to put it together into a chart
+  gather(key = "Parent", value = "value", mom, dad) %>%
+  separate("value", into = c("Name", "Income"))
+### Print out the results
+kable(tidy_dad_mom, format = "markdown")
+```
+
+## Part 2: Joining together CTRP data
+
+```{r show P2 data, echo = FALSE}
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+ccell_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+cmpd_info <- read.csv("CTRP_files/compound_info.csv")
+expt_info <- read.csv("CTRP_files/Experiment_info.csv")
+```
+
+```{r join, echo = FALSE}
+CTRP_total <- AUC_1 %>%
+  bind_rows(AUC_2) %>%
+  inner_join(expt_info, by = c("experiment_id" = "expt_id")) %>%
+  inner_join(ccell_info, by = "master_ccl_id") %>%
+  inner_join(cmpd_info, by = "master_cpd_id") 
+
+### print(CTRP_total)
+### Nevermind, don't print this out, the document is far too long
+```
+
+## Part 2 Q1: Which cancer type has the lowest AUC values to the compound "vorinostat"?
+
+```{r Q1: Which cancer type has the lowest AUC values to the compound "vorinostat"?, echo = TRUE}
+vorinostat_lowest <- CTRP_total %>%
+  filter(cpd_name == "vorinostat") %>%
+  arrange(area_under_curve)
+
+vorinostat_lowest %>%
+  ggplot() +
+  aes(area_under_curve, cancer_type) +
+  labs(title = "Vorinostat area under curve by cancer type", x = "Area under curve", y = "Cancer type") +
+  geom_boxplot()
+
+### print(vorinostat_lowest)
+### Seems like I can't get my boxplot to show up, not sure why. I manually printed out the data using print(vorinostat_lowest) and found that the lowest AUC values are predominantly "Haematopoietic and Lymphoid Tissues"
+```
+
+## Part 2 Q2: Which compound is the prostate cancer cell line 22RV1 most sensitive to?
+
+```{r Q2: Which compound is the prostate cancer cell line 22RV1 most sensitive to?, echo = TRUE}
+cell_sensitive <- CTRP_total %>%
+  filter(ccl_name == "22RV1") %>%
+  arrange(area_under_curve)
+
+cell_sensitive %>%
+  ggplot() +
+  aes(area_under_curve, cpd_name) +
+  labs(title = "22Rv1 sensitive compounds", x = "Area under curve", y = "Compound name") +
+  geom_boxplot()
+
+### print(cell_sensitive) 
+### Getting an error code I copy/pasted below. In the meantime, continuing my brute-force method, I manually printed out the data and found that the highest sensitivity was to Leptomycin B
+#### Error code: File README_files/figure-gfm/Q2: Which compound is the prostate cancer cell line 22RV1 most sensitive to not found in resource path
+#### Error: pandoc document conversion failed with error 99
+#### Execution halted
+#### Not sure how to get past this problem, I'll look at it more when I have more time!
+```
```

Comment:
It seems that your error is not caused by R not being able to find "22RV1" in your data frame like in Ariana's case. Did this error message only appear after you knitted the document or was it from the get go (when you were running individual chunk codes) that this error appeared? Based on my understanding of the error message, R was able to create the plot but the figure was saved in a different directory than your .Rmd file. 

Maybe this link might shed some light on your problem? 
https://community.rstudio.com/t/pandoc-document-conversion-failed-with-error-99/18035

Reply:
Regardless, I applaud your persistence in answering this question by doing a brute-force method. I was able to run my codes and obtain a graph for this question; I also got Leptomycin B as my answer! :)